src/app/components/client/Popover.module.scss

```diff
@@ -1,5 +1,10 @@
 @import "../../tokens";
 
+.underlay {
+  position: fixed;
+  inset: 0;
+}
+
```

Comment:
Weird yeah, it also says

> An underlay is also used to prevent scrolling and interacting with elements outside the popover with a pointer, to avoid unintentially repositioning or closing it.

so that would imply that the `underlayProps` would enforce that, but it looks like it doesn't. But explicitly applying the styles ourselves seem like a fine option to me.